### PR TITLE
Add compose example to build bastion from source

### PR DESCRIPTION
**Pull request by jawabuu**
Enhance docker-compose example to build bastion from source.
We should look at Issue #46 to see why below error occurs when 
SSH_AUDIT_ENABLED=true
```
bastion_1  | Initializing ssh-audit
bastion_1  | - Enabling SSH Audit Logs
bastion_1  | Password: chsh: PAM: Authentication token manipulation error
bastion_1  | FATAL: Failed to initialize
```
@osterman

### CODE REVIEW

**Comment by jawabuu** [comment]
Hey @osterman 
Great to hear from you.
I don't believe I modified the README created generated by README.yaml.
Kindly clarify.

File: examples/compose/README.md [hunk 1]
```diff
@@ -58,14 +58,20 @@ To stop the containers and remove attached volumes, run;
 bastion/examples/compose$ docker-compose down -v
 ```
 
+## Advanced Usage
+
 ### Build from source
-To stop the containers and remove attached volumes, run;
+To build the bastion container from source files in this repo;
```
Comment (maximmi) [inline examples/compose/README.md]: @jawabuu , please make all your changes to the README.yaml, then rebuild. See comments above.

Comment (jawabuu) [reply examples/compose/README.md:64]: Hey @maximmi @osterman , does this apply to the README in the subdirectory?

Comment (max-lobur) [reply examples/compose/README.md:64]: nope, only the root one

**Review by max-lobur — APPROVED** [review]

**Comment by max-lobur** [comment]
Shouldn't this just be `docker-compose -f examples/compose/docker-compose.yml build` though? 
Why does it need another compose file?

**Comment by jonmchan** [comment]
> Shouldn't this just be `docker-compose -f examples/compose/docker-compose.yml build` though? Why does it need another compose file?

The current docker-compose.yml file does not build from source. Executing that command would still pull from hub.docker.com.

That said, I think it is trivial for anyone with decent experience with docker to configure a docker-compose.yml file that builds from source. Additionally, the [`test/docker-compose.yml`](https://github.com/cloudposse/bastion/blob/master/test/docker-compose.yml) now even serves as an example for building from source.

I'd be in favor of just closing this PR.

**Comment by jonmchan** [comment]
Closing due to inactivity. Please reopen this PR if anyone feels this is still important. Thank you!

**Pull request closed without merging**